Mathlib/RingTheory/IntegralClosure/IntegrallyClosed.lean

```diff
@@ -275,3 +275,53 @@ theorem isIntegrallyClosedOfFiniteExtension [IsDomain R] [FiniteDimensional K L]
   (integralClosure_eq_bot_iff L).mp integralClosure_idem
 
 end integralClosure
+
+/-- Any field is integral closed. -/
+instance Field.instIsIntegrallyClosed (K : Type*) [Field K] : IsIntegrallyClosed K :=
```

Comment:
`infer_instance` already finds this (it is `GCDMonoid.toIsIntegrallyClosed`).

Reply:
In this file, `GCDMonoid.toIsIntegrallyClosed` has not been proven yet. But my theorem is a fundamental property of `IsIntegrallyClosed`, and I do not want to involve more content from other files.

Reply:
OK, no problem. It is a `Prop` so no risk of diamonds. Maybe let's add a short about why it is needed.